maple-core/src/easing.rs

```diff
@@ -84,4 +91,98 @@ pub fn quint_inout(t: f32) -> f32 {
     }
 }
 
+// Circular
+
+pub fn circ_in(t: f32) -> f32 {
+    1. - (1. - t.powi(2)).sqrt()
+}
+
+pub fn circ_out(t: f32) -> f32 {
+    (1. - (t - 1.).powi(2)).sqrt()
+}
+
+pub fn circ_inout(t: f32) -> f32 {
+    if t < 0.5 {
+        circ_in(2. * t) / 2.
+    } else {
+        (1. - (-2. * t + 2.).powi(2)).sqrt() / 2.
+    }
+}
+
+// Exponential
+
+pub fn expo_in(t: f32) -> f32 {
+    if t.abs() <= 0.0002 {
```

Comment:
There should be some kind of constant like `EPSILON` instead of hard coding the error margin everywhere.

Reply:
Hey, I'm kinda new and I pressed "Resolve" on all of the above suggested changes; can I somehow have them on my machine locally? Because I don't think they are present, probably because the changes you made live on your branch.

Also, I'm totally happy to further change the code, i.e. change the float-"format" and change the functions.

Reply:
You can go into the "Files Changed" tab at the top and click on "Add suggestion to batch" for each suggestion. You should then be able to commit the suggestions. Finally, run `git pull` on your local machine. Hope this helps.

Reply:
This helped a lot! Thank you!